src/sort/simple-sort-directive.js

```diff
@@ -154,6 +154,26 @@ angular.module('patternfly.sort').directive('pfSimpleSort', function () {
     },
     templateUrl: 'sort/simple-sort.html',
     controller: function ($scope) {
+      var setSortTooltip = function () {
```

Comment:
This name is a bit confusing, more specifically this is the sort direction tooltip.